.clang-format

```diff
@@ -0,0 +1,110 @@
+---
+AccessModifierOffset: -1
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: false
+AlignConsecutiveDeclarations: false
+AlignEscapedNewlines: Left
+AlignOperands:   true
+AlignTrailingComments: true
+AllowAllParametersOfDeclarationOnNextLine: true
+AllowShortBlocksOnASingleLine: false
+AllowShortCaseLabelsOnASingleLine: false
+AllowShortFunctionsOnASingleLine: Inline
```

Comment:
With `true`: https://github.com/JetBrains/kotlin-native/commit/a42e18551cd5f84189551519719f60d2ad8dea04
With `Inline` (diff from `true`): https://github.com/JetBrains/kotlin-native/commit/7ed705072a679d5c13278cdf6c05ae1bc10b4805

Reply:
I like the `Inline` because it basically means that simple functions defined in the header (usually doing simple arithmetics or just forwarding the call) and simple class methods (usually getters and setters) get single-lined, and everything else gets a proper multi-line body.

There's also an option `false` which will make every function definition multi-line.